charts/pulsar/values.yaml

```diff
@@ -1318,7 +1318,7 @@ pulsar_manager:
     # the pulsar manager image relies on these variables, if they are not set the backend will keep crashing
     # however, feel free to overwrite them
     SPRING_CONFIGURATION_FILE: "/pulsar-manager/pulsar-manager/application.properties"
-    PULSAR_MANAGER_OPTS: " -Dlog4j2.formatMsgNoLookups=true"
+    PULSAR_MANAGER_OPTS: "-Xmx400M -Dlog4j2.formatMsgNoLookups=true"
```

Comment:
I guess we could replace `-Xmx400M` with `-XX:MaxRAMPercentage=70.0` so that the memory size adjusts according to the requests / limits. Azul's doc about it: https://docs.azul.com/prime/Heap-Size